apps/meteor/ee/app/models/server/models/LivechatRooms.js

```diff
@@ -38,7 +38,7 @@ LivechatRooms.prototype.setPredictedVisitorAbandonment = function (roomId, willB
 
 LivechatRooms.prototype.findAbandonedOpenRooms = function (date) {
 	return this.find({
-		'omnichannel.predictedVisitorAbandonmentAt': { $lte: date },
+		'omnichannel.predictedVisitorAbandonmentAt': { $lte: new Date(date) },
```

Comment:
```suggestion
		'omnichannel.predictedVisitorAbandonmentAt': { $lte: date },
```

Reply:
Same comment goes [here](https://github.com/RocketChat/Rocket.Chat/pull/25520/files#r874730462)

Reply:
you were right